core/src/test/resources/ExplicitImplicit/basic.source

```diff
@@ -172,15 +305,15 @@ object A {
     implicit val x: List[D.B] = List(new D.B)
   }
 }
-<<< SKIP slick tuple
+<<< slick tuple
 object slick {
   case class Supplier(id: Int, name: String)
   implicit val supplierGetter = (arg: (Int, String)) => Supplier(arg._1, arg._2)
 }
 >>>
 object slick {
   case class Supplier(id: Int, name: String)
-  implicit val supplierGetter: ((Int, String)) => slick.Supplier = (arg: (Int, String)) => Supplier(arg._1, arg._2)
+  implicit val supplierGetter: ((Int, String)) => Supplier = (arg: (Int, String)) => Supplier(arg._1, arg._2)
 }
 <<< NOWRAP package import
 package scala.concurrent {
```

Comment:
Thanks for the code snippets, will add to tests. Forgot about our beloved `locally` :smile:.